lib/licensee/ignore_file.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Licensee
+  class IgnoreFile < ContentFile
+    include Licensee::HashHelper
+    HASH_METHODS = %i[ignored_paths].freeze
+
+    FILENAME = '.licensee-ignore'
+
+    # Default IgnoreFile to maintain bacckward compatability
+    DEFAULTS = %w[xml go].map { |ext| "*.#{ext}" }.freeze
```

Comment:
Should *.gemspec also be in this list? It's [excluded[ in OTHER_EXT_REGEX.

I see licensee has its own .licensee-ignore file, and licensee is probably why gemspec was excluded, so I guess it depends just how absolutely backwards compatible you might want to be (there are a variety of other gems with license in their names that could conceivably be affected, if one ran licensee on them).